runtime/httpserver/server.go

```diff
@@ -137,6 +139,8 @@ func GetMainHandler(options ...func(map[string]http.Handler)) http.Handler {
 	if config.IsDevEnvironment() {
 		defaultRoutes["/explorer/"] = explorer.ExplorerHandler
 		defaultRoutes["/"] = http.RedirectHandler("/explorer/", http.StatusSeeOther)
+		defaultRoutes["/inferences.json"] = InferenceHistoryHandler
+		defaultRoutes["/plugins.json"] = PluginHandler
```

Comment:
This feels weird.  Let's discuss.

Reply:
Per our discussion:

- Move `/inferences.json` and the corresponding `inferenceHistoryHandler` to `explorer/explorer.go` and connect it there as `/explorer/api/inferences` (no `.json` extension).  Wire it to the explorer's own mux.
- Remove `/plugins.json` since we're not using it in the explorer